.mergify/config.yml

```diff
@@ -29,7 +29,6 @@ pull_request_rules:
       - "#review-requested=0"
       - "#changes-requested-reviews-by=0"
       - status-success~=AWS CodeBuild us-east-1
-      - status-success=continuous-integration/travis-ci/pr
```

Comment:
Shouldn't this be replaced with a `status-success` for the GitHub action that builds the image on PR's?

Reply:
Mergify does not allow me to specify conditional checks... I don't want to re-build the image in ALL pr validations (only those that touch the Superchain file)

Reply:
Is it really that bad building it for all PR's? I tend to usually accept the penalty in favor of better validation. 

You could also do the path detection from inside the github action rather than in the workflow file, and just no-op it when needed.

Reply:
> Is it really that bad building it for all PR's? I tend to usually accept the penalty in favor of better validation.

The "penalty" is getting throttled by Docker and having PR validations fail...

But I like where you're going with doing in-action detection... I'll give that a shot.